_dashboards/discover/index-discover.md

```diff
@@ -17,7 +17,7 @@ has_children: true
 
 This tutorial shows you how to use Discover to analyze and understand a sample dataset. At the end of this tutorial, you should be ready to use Discover with your own data.
 
-Before starting this tutorial, make sure you've added the **Sample flight data**. See [Quickstart guide for OpenSearch Dashboards]({{site.url}}{{site.baseurl}}/dashboards/quickstart/) for information about how to get started.
+Before starting this tutorial, make sure you've added the **Sample flight data**. See [Quickstart guide for OpenSearch Dashboards]({{site.url}}{{site.baseurl}}/dashboards/quickstart-dashboards/) for information about how to get started.
```

Comment:
```suggestion
Before starting this tutorial, make sure you've added the **Sample flight data**. See [Quickstart guide for OpenSearch Dashboards]({{site.url}}{{site.baseurl}}/dashboards/get-started/quickstart-dashboards/) for information about how to get started.
```

Reply:
I moved this page to the top level and deleted the `get-started` directory. @vagimeli confirmed that this would've been the only page in the `get-started` section, so I took the opportunity to make the link simpler.